app/lib/logger.js

```diff
@@ -0,0 +1,14 @@
+function info(message) {
+  // eslint-disable-next-line no-console
+  console.log(message);
+}
+
+function error(message) {
+  // eslint-disable-next-line no-console
+  console.error(message);
+}
+
+module.exports = {
+  info,
+  error,
+};
```

Comment:
this reminds me that the other ETL should switch to the standard logger too, is it worth adding an issue to both?